site/content/extend/plugins/webapp/reference.md

```diff
@@ -96,3 +111,80 @@ The theme object has the following properties:
 | mentionHighlightLink | Color of text for mention links in posts |
 | codeTheme | Code block theme, either 'github', 'monokai', 'solarized-dark' or 'solarized-light' |
 
+## Exported Libraries and Functions
+
+The web app exports a number of libraries and functions on the [window](https://developer.mozilla.org/en-US/docs/Web/API/Window) object for plugins to use. We recommend importing as many libraries from the window as possible. Below is a list of the exposed libraries and functions:
+
+
+| Library | Description |
+| -------- | ----------- |
```

Comment:
I'm wondering if it's useful to describe when you'd import a library, and when not? E.g. use `react` library if you do X and Y?

Reply:
Maybe, might be a bit overkill though. If you need a library as a developer you should know why and if you don't need then you can ignore these. Linking out is more then enough IMO